llvm/test/tools/llvm-objcopy/wasm/strip-debug.test

```diff
@@ -1,7 +1,39 @@
-## Test that debug sections (but not linking or names) are stripped with --strip-debug
 # RUN: yaml2obj %s -o %t
-# RUN: llvm-strip --strip-debug %t
-# RUN: obj2yaml %t | FileCheck --implicit-check-not=.debug %s
+# RUN: cp %t %t3
+# RUN: llvm-objcopy --strip-debug %t %t2
```

Comment:
Is it the intention that this test will pass with the updated behavior to actually rewrite the linking and relocation sections? If not, I think we should also explicitly test for the behavior that this test actually adds, i.e. check for the zero-sized renamed sections.

Reply:
Good point.

This test code is copied directly from `llvm/test/tools/llvm-objcopy/ELF/strip-debug.test`, BTW, I should have mentioned that.

I will see if I can update to verify explicitly the new behaviour.